pom.xml

```diff
@@ -1399,5 +1407,19 @@
         </pluginManagement>
       </build>
     </profile>
+    <profile>
```

Comment:
> If there's a good reason why these particular presets are useful

I believe Dima's involvement in this giant review is justification enough for me.